Add QuickJump Links

mcrmonkey opened the pull request:
Added quick jump links to some services so they are easier to find

mcrmonkey committed:
Add QuickJump Links

alulsh commented on README.md:
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
 Make sure 2FA or two step verification is enabled on all of the following accounts:
 
-- [ ] Google
+- [ ] Google - [Quick Jump](https://myaccount.google.com/security/signinoptions/two-step-verification)
```
Hmm could we change to `[Google](https://myaccount.google.com/security/signinoptions/two-step-verification)` instead? I'm not a fan of the "quick jump" phrase. It reminds me of "click here" links - [here's a great overview of why you shouldn't use click here for links](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/).

alulsh commented on README.md:
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
-- [ ] Facebook - enable [Login Approval](https://www.facebook.com/notes/facebook-engineering/introducing-login-approvals/10150172618258920/)
-- [ ] GitHub
-- [ ] Dropbox
+- [ ] Facebook - enable [Login Approval](https://www.facebook.com/notes/facebook-engineering/introducing-login-approvals/10150172618258920/) - [Quick Jump](https://www.facebook.com/settings?tab=security&section=approvals)
```
Change to `Facebook - [enable Login Approval](https://www.facebook.com/settings?tab=security&section=approvals)`

alulsh commented on README.md:
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
-- [ ] GitHub
-- [ ] Dropbox
+- [ ] Facebook - enable [Login Approval](https://www.facebook.com/notes/facebook-engineering/introducing-login-approvals/10150172618258920/) - [Quick Jump](https://www.facebook.com/settings?tab=security&section=approvals)
+- [ ] GitHub - [Quick Jump](https://github.com/settings/security)
```
Change to `[GitHub](https://github.com/settings/security)`

alulsh commented on README.md:
```diff
@@ -52,11 +52,11 @@ A strong complex password is at least 16 characters long (the longer the better)
-- [ ] Dropbox
+- [ ] Facebook - enable [Login Approval](https://www.facebook.com/notes/facebook-engineering/introducing-login-approvals/10150172618258920/) - [Quick Jump](https://www.facebook.com/settings?tab=security&section=approvals)
+- [ ] GitHub - [Quick Jump](https://github.com/settings/security)
+- [ ] Dropbox - [Quick Jump](https://www.dropbox.com/account/#security)
```
Change to `[Dropbox](https://www.dropbox.com/account/#security)`

alulsh commented:
Thanks for this PR and for wanting to improve this checklist, @mcrmonkey! Linking directly to the 2FA settings pages is a good improvement.

I left some line notes on this PR. Once those are addressed I'll gladly merge this PR.

mcrmonkey committed:
Adjusted the links

As per recommendations by @alulsh and info on http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/ the links have been adjusted

mcrmonkey commented:
Links have been updated as per recommendations
The [piece on click here links](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/) was a good read!

alulsh commented:
Thanks again @mcrmonkey! I'm going to merge this now. This will definitely help people configure 2FA on their accounts.
